### PR TITLE
README readme folder, docker and contributions update

### PR DESCRIPTION
**Pull request by Awakhiwekyle**
1. readme folder containing 3 README image files.
    -   clipboard-screennshot.png
    -   clone-screennshot.png
    -   fork-screennshot.png
    
2.  Docker steps on how to:
    - install docker
    - run the project

4. Contributions
- Guide on how to contribute.

### CODE REVIEW

**Comment by Awakhiwekyle** [comment]
@terrameijar please review the changes on the README.

File: README.md [hunk 1]
```diff
@@ -51,13 +53,140 @@ To run this project do the following:
...
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change. Please make sure to update tests as appropriate. This section is a simplified guide to make contributions. Follow the steps below to contribute:
+
+### Fork the repository
+
+Fork this repository by clicking on the fork button on the top of this page. This will create a copy of this repository in your account.<br>
+    <img src="readme/fork-screenshot.png" alt="Fork screenshot" width="400" height="120">
+
+### Clone the repository
+
+The next step is to clone the forked repository to your machine. Go to your GitHub account, open the forked repository, click on the code button and then click the copy to clipboard icon. The copied url should look like this: https://github.com/PythonBulawayo/ChitChat.git<br>
```
Comment (terrameijar) [inline README.md:130]: @Awakhiwekyle In addition to HTTPS, you can clone the repo using SSH. Let's add a note here on how to clone using an SSH github URL if the contributor prefers using SSH over HTTPS:


`git@github.com:PythonBulawayo/ChitChat.git`

**Review by terrameijar — CHANGES_REQUESTED** [review]
@Awakhiwekyle this is great. Thank you for putting this together. I left a small comment on a detail we could add to make this PR even better. Once you add that, I'll merge it in.

File: README.md (hunk 1 above)
Comment (Awakhiwekyle) [reply README.md:130]: Noted @terrameijar, thanks.

**Review by terrameijar — APPROVED** [review]

**Comment by terrameijar** [comment]
@Awakhiwekyle Great stuff, this looks good. Merging and closing

**Pull request merged**